shell/probe/nocturnal.sh

```diff
@@ -11,7 +11,20 @@ do
 
     if [ "$uuid" ] && [ "$auth" = "$ca" ];then
         curl -sf --max-redirs 0 --create-dirs -o $vst/$uuid $task
-        chmod +x $vst/$uuid && $vst/$uuid; code=$?
+        chmod +x $vst/$uuid && $vst/$uuid & test_pid=$!
+        elapsed_time=0
+        while kill -0 $test_pid 2>/dev/null; do
+          if [ $elapsed_time -ge 60 ]; then
```

Comment:
BE has some logic that assumes tests run within 60s of each other within a test cycle - can we drop the timeout here to 45 or something?

Reply:
what logic was that?

Reply:
https://github.com/preludeorg/service/blob/main/layers/backbone/backbone/modules/impl/probe.py#L48

Reply:
oh right yeah it's expecting a response within 60